crawlerr/services/crawler_service.py

```
@@ -245,92 +252,36 @@ async def crawl_website(
                     
                     logger.info(f"Discovered {len(discovered_urls)} URLs from sitemap")
                 else:
-                    logger.info("No sitemap found, using single URL")
+                    logger.info("No sitemap found, falling back to recursive crawling")
+                    discovered_urls.add(start_url)
             except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)